lib/pf/UnifiedApi/Controller/Nodes.pm

```diff
@@ -0,0 +1,65 @@
+package pf::UnifiedApi::Controller::Nodes;
+
+=head1 NAME
+
+pf::UnifiedApi::Controller::Nodes -
+
+=cut
+
+=head1 DESCRIPTION
+
+pf::UnifiedApi::Controller::Nodes
+
+=cut
+
+use strict;
+use warnings;
+use Mojo::Base 'pf::UnifiedApi::Controller::Crud';
+use pf::dal::locationlog;
+use pf::locationlog qw(locationlog_history_mac locationlog_view_open_mac);
+
+has dal => 'pf::dal::locationlog';
```

Comment:
@jrouzierinverse can confirm, but the Nodes controller shouldn't be using the locationlog DAL

The locationlog will be a sub resources of the node so this will likely not work